inc/class-upgrade.php

```diff
@@ -547,4 +520,40 @@ private function upgrade_70() {
 		global $wpdb;
 		$wpdb->query( "UPDATE $wpdb->usermeta SET meta_key = 'wpseo_noindex_author' WHERE meta_key = 'wpseo_excludeauthorsitemap'" );
 	}
+
+	/**
+	 * Retrieves the option value directly from the database.
+	 *
+	 * @param string $option_name Option to retrieve.
+	 *
+	 * @return array|mixed The content of the option if exists, otherwise an empty array.
+	 */
+	protected function get_option_raw( $option_name ) {
```

Comment:
Suggestion: You are getting an options from the database thus `get_option_from_database` would be nicer.